build.sh

```diff
@@ -3,19 +3,5 @@
 set -e
 set -u
 
-PROJECT_ROOT="$(cd "$( dirname "$0" )"; pwd)"
-
-# Verify that repository has been cloned with submodules
-
-cd "$PROJECT_ROOT/lib/libelectronic-id"
-
-[[ -e README.md ]] || { echo "FAIL: libelectronic-id submodule directory empty, did you 'git clone --recursive'?"; exit 1; }
-
-# Build everything
-
-cd "$PROJECT_ROOT"
```

Comment:
`cd "$PROJECT_ROOT"` is needed when the script is run from a different location than project root. I'll merge this as-is, but we should add it back later.